README.md

```diff
@@ -286,7 +279,7 @@ Finally, we can move to part describing the new cool features we bring with `uni
 * Conversions between `Either` and `Maybe` like `rightToMaybe` and `maybeToLeft`
   with clear semantic.
 * `using(Reader|State)[T]` functions as aliases for `flip run(Reader|State)[T]`.
-* [`One` type class](https://github.com/serokell/universum/blob/master/src/Containers.hs#L473)
+* [`One` type class](https://github.com/kowainik/universum/blob/master/src/Containers.hs#L473)
```

Comment:
**Note for future**: I discovered that in such links in GitHub readme it's possible to use relative links (like `src/Containers.hs` (btw, this link is broken anyway)) so in future we should remake all links.